### DISCUSSION TITLE
What is yhat1

### DISCUSSION THREAD

**Discussion by DaToKi** [ourownstory/neural_prophet#201] [Q&A - get help using NeuralProphet] [answered]
![2020-12-09_101346](https://user-images.githubusercontent.com/45854141/101609451-44e56b00-3a07-11eb-9544-8deb055582b7.jpg)
In the created df forecast there is ds and y.

What are the values of yhat1?

**Comment by HansikaPH** [comment, marked as answer]
Hi @DaToKi, yhat1 values are the predictions from the model. In this case, since you are forecasting into the future, the true y values are not available, but the forecasts(yhat1) are available from the model.

**Reply by DaToKi** [reply]
Thanks!

**Reply by wangzhongqing** [reply]
hi, can you save model? @HansikaPH

**Comment by ourownstory** [comment]
@DaToKi if you do not specify any `n_lags`,  `yhat1` is identical to the returned `yhat` in fbprophet. 

if `n_lags > 0` is specified, the number 'n' after `yhat'n'` refers to the 'n'-th step forecasted into the future, up to `n_forecasts` steps.

**Reply by DaToKi** [reply]
Thanks!

**Reply by wangzhongqing** [reply]
hi, can you save model? @ourownstory

**Reply by ourownstory** [reply]
@wangzhongqing currently no, but you could implement #27 yourself

**Reply by nachi-lieder** [reply]
Hi, thanks, when predicting 5 days ahead , i get 5 yhats, but some are NaN. how does that work? which one should I use?
